src/olympia/scanners/admin.py

```diff
@@ -368,3 +370,29 @@ def matched_results_link(self, obj):
         return format_html('<a href="{}">{}</a>', url, count)
 
     matched_results_link.short_description = 'Matched Results'
+
+
+@admin.register(ScannerQueryResult)
+class ScannerQueryResultAdmin(admin.ModelAdmin):
+    pass
+
+
+@admin.register(ScannerRule)
+class ScannerRuleAdmin(AbstractScannerRuleAdminMixin, admin.ModelAdmin):
+    pass
+
+
+@admin.register(ScannerQueryRule)
+class ScannerQueryRuleAdmin(AbstractScannerRuleAdminMixin, admin.ModelAdmin):
```

Comment:
This and the one above are not very useful right now as they don't expose a way to trigger an action yet, and what they are displaying isn't necessarily what we want, but I needed them for easier debugging.